package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "build": "rimraf dist && ncc build src/index.ts && ts-node bin/generate-docs.ts",
     "lint": "eslint . --ignore-pattern='!.eslintrc.js' --ext .ts,.js",
-    "test": "echo no tests yet"
+    "test": "yarn tsc && jest"
```

Comment:
Running `tsc` because I'm looking at the JS file for the action runner. I tried using `ts-node` directly but it was complaining about not finding `remote-git-tags` type defs. This might be the way to go anyway if we decide to move that runner into a separate library later.

Reply:
When you tried to use ts-node directly were you requiring it into jest or passing as part of the command?

Yeah this is for sure fine for now, just interested to know more

Reply:
I was using `ts-node` to call the runner via the `exec` command. I didn't add it to Jest in any particular way. Didn't seem necessary based on error it was returning.